picamctl/picamctl.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+# akseidel 02/06/2021
+# A script automating the steps to run showmewencam's "camera-ctl".
+# The script will pause for continue or exit if run with an argument.
+
+# edit this next line to be the correct text pattern for your computer.
+portnamepat="tty.usbmodem" #typical for showmewebcam on Appl OS X
+#portnamepat="ttyACM" #probably typical for showmewebcam on Linux
+#portnamepat="tty."
```

Comment:
You should auto discover this, based on the platform you are on.

Reply:
Ok, I will work on that. I have a Raspberry Pi 4 I can use for testing the Linux side.

Reply:
Auto discover is now implemented.